files/laravel/README.md

```diff
@@ -188,21 +188,22 @@ protected $proxies = '*';
 // ...
 ```
 
-If you want to be more restrictive and only trust the reverse proxy used in the server, you must first obtain its address. The easiest way to get it is to read the Nginx access log (if you take a look at it, you will notice that all requests comes from the same IP address, the reverse proxy's one):
+If you want to be more restrictive and only trust the reverse proxy used in the server, you must first obtain its IP address. The easiest way to get it is to read the Nginx access log (if you take a look at it, you will notice that all requests comes from the same IP address, the reverse proxy's one):
 
 ```bash
 $ # Takes the last line of the Nginx access log and
-$ # outputs the "client" address, in our case the reverse proxy's
+$ # outputs the "client" IP address, in our case the reverse proxy's
 $ tail -n 1 ~/www/logs/nginx_access.log | cut -d ' ' -f 1
 ```
+This command will output the reverse proxy private IP address. Let us assume that it is ``172.18.0.6`` but make sure to run this command yourself and use its output afterwards, the address may be different on your server for any reason whatsoever.
```

Comment:
I'd put the real example in the code block.

```console
% tail -n 1 ...
172.18.0.6
```

People don't read.

Reply:
Haha, true. Changing this now.